app/src/main/java/org/covidwatch/android/presentation/home/HomeViewModel.kt

```diff
@@ -48,8 +48,6 @@ class HomeViewModel(
     private val interactedWithInfectedObserver = Observer<Boolean> { hasPossiblyInteractedWithInfected ->
```

Comment:
Isn't it nicer for these bits of logic to live next to each other? It should be one banner or the other

Reply:
If current user is tested positive should depend on `Observer` that listens to `contact_event` changes in db

This `if (hasPossiblyInteractedWithInfected && !isUserTestedPositive) {` makes sure those banners will not appear at the same time